Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1275,6 +1275,18 @@ def lhegensim2017(fragment,howMuch):
     '--procModifiers': 'premix_stage2',
     '--era'          : 'Run2_2016'
     }
+# Specifying explicitly the --filein is not nice but that was the
+# easiest way to "skip" the output of step2 (=premixing stage1) for
+# filein (as it goes to pileup_input). It works (a bit accidentally
+# though) also for "-i all" because in that case the --filein for DAS
+# input is after this one in the list of command line arguments to
+# cmsDriver, and gets then used in practice.
+digiPremixLocalPileup = {
```

Comment:
I assume you mean that this works for the single workflow but not necessarily when injecting the production of the overall sample. I think this is a question for PdmV, how is this managed there?

Reply:
@davidlange6 
I do not think these kinds of premix workflows can run as a relval, because in a taskchain all the output datasets from the workflow steps are being produced at the same time (and not one after the other). So the mixing step will search for an input PU dataset that is still in production. In addition to this, I am not sure if it is possible to specify in WMAgent an input PU taken from a previous step of the workflow.

But I think these workflows are only needed for testing a PR or to run in IB, right?

Reply:
> But I think these workflows are only needed for testing a PR or to run in IB, right?

That (PR+IB) is indeed my intention. I'd think the current separate stages approach works indeed better for RelVal (and production).

Reply:
Ok, let's take this anyway in mind